flash_update/views.py

```diff
@@ -14,11 +14,17 @@
 from .models import (
     FlashUpdate,
     FlashGraphicMap,
-    FlashAction
+    FlashAction,
+    DonorGroup,
+    Donors,
+    FlashUpdateShare
 )
 from .serializers import (
     FlashUpdateSerializer,
-    FlashGraphicMapSerializer
+    FlashGraphicMapSerializer,
+    DonorGroupSerializer,
+    DonorsSerializer,
+    ShareFlashUpdateSerializer
```

Comment:
Let's always add `,` for the last item in the list as well. This creates a better diff for future changes.